code/src/store/store.ts

```diff
@@ -12,7 +13,7 @@ export const store = configureStore({
     [appApi.reducerPath]: appApi.reducer
   },
   middleware: (getDefaultMiddleware) =>
-    getDefaultMiddleware().concat(appApi.middleware)
+    getDefaultMiddleware().concat(appApi.middleware).concat(errorMiddleware)
```

Comment:
Maybe move all the middlewares to a separate const and then `getDefaultMiddleware().concat(middlewares)`. What do you think?

Reply:
Big thank you to you. Is is really worth to move it to separate variable. I did it like you have suggested